### DISCUSSION TITLE
Submodules

### DISCUSSION THREAD

**Discussion by knotFF** [ProjectBorealis/UEGitPlugin#106] [General]
Is there a way to work with submodules?

**Comment by ameaninglessname** [comment]
It should already be supported

**Reply by knotFF** [reply]
Thanks for the reply.
Could you demonstrate how its possible to push content to a submodule?
In the example bellow, I have created a git project for 5.2 and added a Plugin as submodule.
Pushing the main project is fine, but pushing content to the submodule is not, furthermore I see no way of pushing the pointer of the submodule.

![20231024_083125](https://github.com/ProjectBorealis/UEGitPlugin/assets/20506400/f4dae905-b145-4370-a109-9f00f0ca8eb6)
